ui/components-react/src/components-react/newEditors/values/Metadata.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Value types supported by editor system.
+ * @beta
+ */
+export type ValueType =
+  | "string"
+  | "number"
+  | "bool"
+  | "date"
+  | "enum"
+  | "instanceKey";
+
+/**
+ * Additional metadata that is used along side value to determine applicable editor.
```

Comment:
do we want to mention "editor" in docs of these types? I could see them being used in non-editing contexts, e.g. read-only property rendering use cases

Reply:
Removed `editor` from comment